packages/anima-cli/src/helpers/token.ts

```diff
@@ -0,0 +1,18 @@
+import { isDebug } from "./debug";
+
+export class TokenError extends Error {
+  constructor(message: string) {
+    super('[TokenError]: ' + message);
+  }
+}
+
+export const getToken = (argv: Record<string, unknown>) => {
+  const token = (argv.token ??
+    process.env.STORYBOOK_ANIMA_TOKEN ??
```

Comment:
I think we also should change the token's var name, to something less specific to storybook. as it is a Team token after all.

```suggestion
    process.env.ANIMA_TEAM_TOKEN ??
```